### ISSUE TITLE
Sidebars cannot be used with keyboard only

### ISSUE THREAD

**Issue by can3p** [LightTable/LightTable#1106]
If I open Workspace or Connections sidebar I have no way to put keyboard focus there and to choose anything with keyboard.

**Comment by kenny-evitt** [comment]
I keep wishing for this, but it's pretty low on the priority totem pole now, so I added it to [the feature wishlist wiki page](https://github.com/LightTable/LightTable/wiki/Feature-wishlist).

**Comment by 1j01** [comment]
Still a problem in LT 0.8.1

**Comment by kenny-evitt** [comment]
@1j01 Would you like to implement this?

**Comment by 1j01** [comment]
@kenny-evitt If I could actually get Light Table's connection feature(s) to work (with Node.js and CoffeeScript), then I would happily work on some of these issues I've opened / commented on, but as it is, Light Table isn't really useful to me. \<sad face emoji that doesn't exist\>
